src/components/Term.js

```diff
@@ -8,15 +9,22 @@ const Term = ({ children, dispatch, id }) => {
     e.preventDefault()
     dispatch(showTerm(id))
   }
+  const readable = lowerCase(id)
 
   return (
-    <a className="border-bottom-dotted" href="#!" onClick={handler}>
+    <a
+      aria-label={`show ${readable} in the glossary`}
```

Comment:
since `readable` only used once, i would just sub in `lowerCase(id)` here to avoid the abstraction